ps_socialfollow.php

```diff
@@ -187,11 +202,32 @@ public function getConfigFieldsValues()
     {
         $result = [];
         foreach (static::SOCIAL_NETWORKS as $social) {
-            $result['blocksocial_' . $social] = Configuration::get('BLOCKSOCIAL_' . strtoupper($social));
+            if (!empty(Configuration::get('BLOCKSOCIAL_' . strtoupper($social)))) {
+                $this->upgradeConfiguration('BLOCKSOCIAL_' . strtoupper($social));
+            }
+            foreach ($this->context->controller->getLanguages() as $lang) {
+                $id_lang = $lang['id_lang'];
+                $conf = Configuration::get('BLOCKSOCIAL_' . strtoupper($social), $id_lang);
+                $result['blocksocial_' . $social][$id_lang] = $conf;
+            }
         }
         return $result;
     }
 
+    protected function upgradeConfiguration($name)
+    {
+        $conf = Configuration::get($name);
+        if (!empty($conf) && !is_array($conf)) {
+            $conf_localized = [];
+            foreach ($this->context->controller->getLanguages() as $lang) {
+                $conf_localized[$lang['id_lang']] = $conf;
+            }
+            Configuration::updateValue($name, $conf_localized);
+            $conf = $conf_localized;
+        }
+        return $conf;
```

Comment:
Usually we insert a blank line before the `return` (I think PHP CS Fixer applies this on the main repository)